requirements.txt

```diff
@@ -13,5 +13,6 @@
 #   limitations under the License.
 
 
-grpcio-tools==1.38.1
-th2-grpc-sim~=3.1.3
+grpcio-tools==1.56.0
```

Comment:
I don't think that we need it here. I will be installed as transitive dependency